packages/panels/src/components/nav-bar/NavBarUserButton.module.css

```diff
@@ -28,6 +28,17 @@
   --swui-button-border-radius: 16px;
   --swui-button-border-radius-icon-only: 16px;
 
+  &:not(.initials) {
+    @media (max-width: 512px) {
+      .iconWrapper {
```

Comment:
This is kind of hacky, and might break unnoticed when refactoring :/ 

Isn't it easier to just leave it up to the app to pass label and/or icon, properly?
That would require React listening for media queries, but on the other hand, it won't break. But on the first hand, what is the purpose of this component then.. So.. Idk :D

Reply:
I changed the implementation